tidb-cloud/built-in-monitoring.md

```diff
@@ -19,7 +19,11 @@ To view the metrics on the Monitoring page, take the following steps:
 
 2. Click the **Monitoring** tab.
 
-## Monitoring metrics
+## Metrics Retention Policy
+
+All monitoring data are kept for 2 days.
+
```

Comment:
Question: How to view yesterday's monitoring data?

Reply:
change the time picker, users will see the time picker on UI. It's obvious.